mmf/models/interfaces/mmbt.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import os
+import tempfile
+from pathlib import Path
+from typing import Type, Union
+
+import torch
+import torchvision.datasets.folder as tv_helpers
+from PIL import Image
+from torch import nn
+
+from mmf.common import typings as mmf_typings
+from mmf.common.sample import Sample, SampleList
+from mmf.models.base_model import BaseModel
+from mmf.utils.build import build_processors
+from mmf.utils.download import download
+
+MMBT_GRID_HM_CONFIG_PATH = Path("projects/hateful_memes/configs/mmbt/defaults.yaml")
+ImageType = Union[Type[Image.Image], str]
+PathType = Union[Type[Path], str]
+BaseModelType = Type[BaseModel]
+
+
+class MMBTGridHMInterface(nn.Module):
```

Comment:
I would extend this class from `MMBT` and add only the extra methods. 

Ideally since every dataset can have different implementation for `classify` specific to a model, the better way to do it will be move dataset specific parts to the dataset classes and model specific parts to the model itself. That will help get rid of these extra interface class. We should discuss more about this design. I will approve this now since the PR needs to land urgently.

Reply:
Sure. Let's discuss this separately outside of this PR.